code-of-conduct.md

```diff
@@ -0,0 +1,58 @@
+## Kubernetes Community Code of Conduct
```

Comment:
Can't this file just be:

See the [kubernetes project code of conduct](https://github.com/kubernetes/kubernetes/blob/master/code-of-conduct.md).